airflow/models.py

```diff
@@ -1594,22 +1656,28 @@ def signal_handler(signum, frame):
                 Stats.gauge(
                     'ti_status',
                     value=0,
-                    tags=[
+                tags=[
```

Comment:
is this tabbed correctly?

Reply:
Whoops!  It is not.  Fixed.  (batching changes up for testing - will be in the next commit)